src/gui/accountsettings.cpp

```diff
@@ -62,6 +62,28 @@
 
 namespace {
 constexpr auto propertyFolderInfo = "folderInfo";
+
+bool showEnableE2eeWithVirtualFilesWarningDialog()
+{
+    QMessageBox e2eeWithVirtualFilesWarningMsgBox;
+    e2eeWithVirtualFilesWarningMsgBox.setText("End-to-End Encryption with Virtual Files");
```

Comment:
@FlexW Are the QMessageBox texts always considered translatable? I just didn't find anything about that in the QMessageBox's documentation page, while I don't see a `tr()` wrapper for the string you pass here either.

Reply:
D'oh, yes, stupid me. It's missing tr() everywhere here.

Reply:
thanks @allexzander. I forgot that completely.